galata/test/documentation/media_files.test.ts

```diff
@@ -0,0 +1,65 @@
+// Copyright (c) Jupyter Development Team.
+// Distributed under the terms of the Modified BSD License.
+
+import { expect, galata, test } from '@jupyterlab/galata';
+import { filterContent } from './utils';
+
+test.use({
+  autoGoto: false,
+  mockState: galata.DEFAULT_DOCUMENTATION_STATE,
+  viewport: { height: 720, width: 1280 }
+});
+
+test.describe('Media Files', () => {
+  test('should open and display audio file', async ({ page }) => {
+    await galata.Mock.freezeContentLastModified(page, filterContent);
+    await page.goto();
+    await page.addStyleTag({
+      content: `.jp-LabShell.jp-mod-devMode {
+        border-top: none;
+      }`
+    });
+
+    await page.sidebar.setWidth();
```

Comment:
If you do not plan to add snapshots here these could be removed in both tests.

Reply:
yeah good catch!